blueocean-web/src/main/js/init.jsx

```diff
@@ -43,6 +43,11 @@ exports.initialize = function (oncomplete) {
     const Extensions = require('@jenkins-cd/js-extensions');
     jenkinsMods.export('jenkins-cd', 'js-extensions', Extensions);
 
+    // Create and export a shared instance of the core
+    // js module
+    const corejs = require('@jenkins-cd/blueocean-core-js');
+    jenkinsMods.export('jenkins-cd', 'blueocean-core-js', corejs);
+
```

Comment:
Fixed in 61f0b7c.  @tfennelly if you could double-check the changes there I would appreciate it. It did restore the ToastService to proper working order.

One more question too: when building the new `js-extensions` I noticed that the test "js-builder plugin test -> test yamlToJSON" is failing. I tried adjusting the destination path to "../target..." but I still got an error when requiring the file, even though it was there. Have you seen this, perhaps know the fix? It's broken on master too.